addons/fortify/XEH_postInit.sqf

```diff
@@ -63,3 +64,18 @@ GVAR(objectRotationZ) = 0;
         [_object, 0, ["ACE_MainActions"], _removeAction] call ACEFUNC(interact_menu,addActionToObject);
     };
 }] call CBA_fnc_addEventHandler;
+
+[QGVAR(setMarkerVisible), {
+    params ["_object"];
+    [
+        {!isNull player},
+        {
+            params ["_object"];
+            if (GVAR(markObjectsOnMap) isEqualTo 1 || {[_object getVariable QGVAR(objectSide), side group player] call BIS_fnc_sideIsEnemy}) then {
```

Comment:
Might prefer [`side1 getFriend side2 < 0.6`](https://community.bistudio.com/wiki/getFriend), overhead of BIS functions is usually not great.